### ISSUE TITLE
On SSR effect starts working only when cursor goes off container

### ISSUE THREAD

**Issue by evgenii-code** [Aminerman/vue-kinesis#51]
I'm using this plugin in Nuxt with SSR.
if mouse cursor is inside kinesis container on initial page load, the mouse parallax effect starts working only when the cursor goes off container and goes back.
As a container i'm using whole layout, so cursor have to go off viewport to effect starts working.
if plugin registered with "mode: 'client'", i get vue warn "The client-side rendered virtual DOM tree is not matching server-rendered content."

i'm kinda newbie, so maybe i'm doing something wrong.

**Comment by evgenii-code** [comment]
For some reason I can't install vue-kinesis in codesandbox
So here is a little repo with my problem https://github.com/evgenii-code/vue-kinesis-test

**Comment by Aminerman** [comment]
Hello! Thank you for providing a repository. That's quite helpful!
Hmm well, I did test your repository and here is what happened: 

1. I tested it the first time and I indeed had the issue you described.
2. I tried to install a previous version of vue-kinesis (1.1.7) and it worked fine.
3. I reinstalled 1.2.1 again and it worked fine.
4. I cloned your repository again in a new folder, and everything worked fine.
5. I tried to test it on Codesanbox, but the nuxt app breaks when I install a dependency.

So I didn't really have a chance to understand why the issue occured in the first place as it seems to be working fine. 

I'll keep the issue open for now. I'll try again later.
